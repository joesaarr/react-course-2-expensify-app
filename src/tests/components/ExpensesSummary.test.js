import React from 'react';
import { shallow } from 'enzyme';
import { ExpensesSummary } from '../../components/ExpensesSummary';

test('Should render ExpensesSummary correctly', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={23} expensesTotal={150234500} />);
  expect(wrapper).toMatchSnapshot();
});

test('Should render ExpensesSummary with one expense correctly', () => {
  const wrapper = shallow(<ExpensesSummary expenseCount={1} expensesTotal={54632}/>);
  expect(wrapper).toMatchSnapshot();
});