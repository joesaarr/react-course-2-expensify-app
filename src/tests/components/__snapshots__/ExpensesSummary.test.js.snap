// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpensesSummary correctly 1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1
      className="page-header__title"
    >
      Viewing 
      <span>
        23
      </span>
       
      <span>
        expenses
      </span>
       totalling 
      <span>
        €1,502,345.00
      </span>
    </h1>
    <div
      className="page-header__actions"
    >
      <Link
        className="button"
        to="/create"
      >
        Add expense
      </Link>
    </div>
  </div>
</div>
`;

exports[`Should render ExpensesSummary with one expense correctly 1`] = `
<div
  className="page-header"
>
  <div
    className="content-container"
  >
    <h1
      className="page-header__title"
    >
      Viewing 
      <span>
        1
      </span>
       
      <span>
        expense
      </span>
       totalling 
      <span>
        €546.32
      </span>
    </h1>
    <div
      className="page-header__actions"
    >
      <Link
        className="button"
        to="/create"
      >
        Add expense
      </Link>
    </div>
  </div>
</div>
`;
